.github/workflows/security.yml

```diff
@@ -0,0 +1,18 @@
+name: Security
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Dependencies audit
+      run: yarn audit --groups dependencies --level high
```

Comment:
This is going to ignore issues in dev dependencies and issues with level below high.